src/util/check.h

```diff
@@ -46,4 +46,12 @@ class NonFatalCheckError : public std::runtime_error
 #error "Cannot compile without assertions!"
 #endif
 
+/** Identity function. Abort if the value compares equal to zero */
+template <typename T>
+T&& Assert(T&& value)
```

Comment:
In commit "util: Add Assert identity function" (fa54cf107319609ff44de960b987bd09b46945d0)

Thinking about this, I think it would be better to change signature to `T Assert(T&& value)` to avoid a possibility of returning a reference to a temporary. Contrived example but I think `p` would be an invalid reference here returning `T&&` instead of `T`:

```c++
std::unique_ptr<int>&& p = Assert(MakeUnique<int>(5));
```